private-templates-service/adaptivecards-templating-service/src/util/templateutils.ts

```diff
@@ -0,0 +1,96 @@
+import { ITemplate, ITemplateInstance } from "../models/models";
+
+/**
+ * @function
+ * Returns a ITemplate object with only the latest version instance. 
+ * @param template 
+ */
+export function getMostRecentTemplate(template : ITemplate): ITemplate {
+  if (!template.instances || template.instances.length === 0) return template; 
+  let latestInstance : ITemplateInstance[] = [getMostRecentVersion(template)!];
+  template.instances = latestInstance;
```

Comment:
This will modify the object passed in (objects pass by reference) - is this intentional?

Reply:
Yes, I'm currently using the function to edit the passed in template object's instances to only include the latest version!

Reply:
In that case, these functions shouldn't return anything and be named something like `updateTemplateToLatestInstance` -> just to make sure that the usage of the function is clear, and the comment should match the functionality

Reply:
Fixed, thanks!